platform-ajax/src/main/java/com/softicar/platform/ajax/resource/AjaxResourceUrlParser.java

```diff
@@ -0,0 +1,42 @@
+package com.softicar.platform.ajax.resource;
+
+import com.softicar.platform.common.io.resource.hash.ResourceHash;
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+/**
+ * Extracts resource hash and resource ID from resource URLs.
+ *
+ * @author Oliver Richers
+ */
+public class AjaxResourceUrlParser {
+
+	private static final Pattern HASH_PATTERN = Pattern.compile(".*\\?resourceHash=([0-9a-fA-F]+).*");
```

Comment:
- Should we support URLs in which `resourceHash` is _not_ the first parameter? Like so: `\\?` --> `[\\?&]`
- I think we should reuse the constant `AjaxResourceUrl.HASH_PARAMETER_NAME` here.

Reply:
did so